build-system/tasks/runtime-test/helpers.js

```diff
@@ -255,20 +255,12 @@ function karmaRunStart_() {
  * @return {!Promise<number>} exitCode
  */
 async function runTestInSauceLabs(config) {
-  const browsers = {stable: [], beta: []};
-  for (const browserId of config.browsers) {
-    browsers[
-      browserId.toLowerCase().endsWith('_beta') ? 'beta' : 'stable'
-    ].push(browserId);
-  }
-
-  if (argv.stable) {
-    browsers.beta = [];
-  } else if (argv.beta) {
-    browsers.stable = [];
-  }
+  const browsers = config.browsers.map(name => name.toLowerCase());
 
-  return await runTestInBatches_(config, browsers);
+  return await runTestInBatches_(config, {
+    stable: argv.beta ? [] : browsers.filter(name => name.endsWith('_beta')),
+    beta: argv.stable ? [] : browsers.filter(name => !name.endsWith('_beta')),   
```

Comment:
For better readability, how about making `stable` depend on `argv.stable` and vice versa? In other words:

```js
    stable: argv.stable ? browsers.filter(name => !name.endsWith('_beta')) : [],
    beta: argv.beta ? browsers.filter(name => name.endsWith('_beta')) : [],
```

Reply:
This isn't quite the case though, since `argv.stable !== !argv.beta`. In the form you suggested, both would be empty if neither flag is specified. Instead, we want to exclude one set if the flag for the other set is present.

Reply:
> Thanks! Do you know why we're testing on Chrome twice? https://travis-ci.org/ampproject/amphtml/jobs/657948850#L2400-L2402

I do not